src/oc/web/stores/activity.cljs

```diff
@@ -147,6 +147,29 @@
         next-board-data (assoc board-data :fixed-items next-fixed-items)]
     (assoc-in db board-key next-board-data)))
 
+(defmethod dispatcher/action :activity-move
+  [db [_ activity-data org-slug board-data is-all-posts]]
+  (let [fixed-activity-data (merge activity-data {:board-slug (:slug board-data)
+                                                  :board-name (:name board-data)
+                                                  :board-uuid (:uuid board-data)})]
+    (if is-all-posts
```

Comment:
Because of the all posts refactor, I think it is easier to update both keys (:all-posts and :board-slug) in the app state.  That way the `is-all-posts` boolean can be removed.  Also when I merge I don't have as many conflicts when using `dispatcher/activity-key`.